src/main/scala/com/gu/hmac/HMACHeaders.scala

```diff
@@ -18,33 +18,42 @@ sealed trait HMACError extends NoStackTrace
 case class HMACInvalidTokenError(message: String) extends HMACError
 case class HMACInvalidDateError(message: String) extends HMACError
 
-case class HMACToken(value: String)
 
+object HTTP extends Enumeration {
```

Comment:
Should we wait until we actually need this code to bring it in? Also, I think 'OPTION' should be 'OPTIONS'